WalletWasabi.Gui/Tabs/SettingsView.xaml

```diff
@@ -46,6 +46,34 @@
           </StackPanel>
         </controls:GroupBox>
 
+        <controls:GroupBox Title="Anonymity" TextBlock.FontSize="16" Padding="10" Margin="0 5 10 5">
+          <StackPanel Spacing="10" Orientation="Vertical">
+            <StackPanel Margin="0 10">
+              <TextBlock>Some</TextBlock>
+              <TextBox Text="{Binding SomePrivacyLevel}"/>
+            </StackPanel>
+
+            <StackPanel Margin="0 10">
+              <TextBlock>Fine</TextBlock>
+              <TextBox Text="{Binding FinePrivacyLevel}"/>
+            </StackPanel>
+
+            <StackPanel Margin="0 10">
+              <TextBlock>Strong</TextBlock>
+              <TextBox Text="{Binding StrongPrivacyLevel}"/>
+            </StackPanel>
+          </StackPanel>
+        </controls:GroupBox>
+
+        <controls:GroupBox Title="Other settings" TextBlock.FontSize="16" Padding="10" Margin="0 5 10 5">
+          <StackPanel Spacing="10" Orientation="Vertical">
+            <StackPanel Margin="0 10">
+              <TextBlock>Dust Threshold</TextBlock>
```

Comment:
```suggestion
              <TextBlock>Dust Threshold (BTC)</TextBlock>
```

Reply:
I think also some explanation here would be good.

"Under the dust threshold coins aren't appearing in the coin lists."